lisp/init.ros

```diff
@@ -1,20 +1,180 @@
 #!/bin/sh
 #|-*- mode:lisp -*-|#
-#|Create new ros script
+#|Create a new ros script. Not to be confused with init.lisp
```

Comment:
This line is showed when you type `ros help`.
I think It shouldn't be an advice for roswell developper.